platforms/software/build-init/src/main/java/org/gradle/buildinit/plugins/internal/AbstractBuildGenerator.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2018 the original author or authors.
```

Comment:
❓ Should the new files use the current year for copyright?

Reply:
It should indeed. Not sure what happened there.

Reply:
Fixed